cmd/fireacme/cli/node.go

```diff
@@ -95,7 +95,7 @@ func nodeFactoryFunc(flagPrefix, kind string) func(*launcher.Runtime) (launcher.
 		nodeDataDir := replaceNodeRole(kind, mustReplaceDataDir(sfDataDir, viper.GetString(flagPrefix+"data-dir")))
 
 		readinessMaxLatency := viper.GetDuration(flagPrefix + "readiness-max-latency")
-		debugDeepMind := viper.GetBool(flagPrefix + "debug-deep-mind")
+		debugFirehose := viper.GetBool(flagPrefix + "debug-deep-mind")
```

Comment:
This feels bad now, and the flag needs to change. Also you will need to validate this works correctly, I can help with that.

Reply:
`debug-firehose-logs`